docs/source/guides/data.rst

```diff
@@ -230,15 +226,21 @@ needs to wrap the DataLoaders with `CombinedLoader`.
         loader_a = DataLoader()
         loader_b = DataLoader()
         loaders = {"a": loader_a, "b": loader_b}
-        combined_loaders = CombinedLoader(loaders, "max_size_cycle")
+        combined_loaders = CombinedLoader(loaders, mode="max_size_cycle")
         return combined_loaders
 
 
-Test with additional data loaders
-=================================
-You can run inference on a test set even if the :func:`~pytorch_lightning.core.Lightning.LightningModule.test_dataloader` method hasn't been
-defined within your :class:`~pytorch_lightning.core.Lightning.LightningModule` instance. For example, this would be the case if your test data
-set is not available at the time your model was declared. Simply pass the test set to the :func:`~pytorch_lightning.trainer.trainer.Trainer.test` method:
+    def validation_step(self, batch, batch_idx):
+        batch_a = batch["a"]
+        batch_b = batch["b"]
+
+
+Evaluate with Additional DataLoaders
+====================================
+
+You can evaluate your models using additonal dataloaders even if the dataloader specific hooks hasn't been defined within your
+:class:`~pytorch_lightning.core.lightning.LightningModule` instance. For example, this would be the case if your test data
```

Comment:
```suggestion
:class:`~pytorch_lightning.core.lightning.LightningModule`. For example, this would be the case if your test data
```

The definition of hooks/methods is in the class